#!/usr/bin/python3

def raise_exception_msg(message=""):
    """
    raises a NameError with a message
    args:
    message (str): a string to pass as a message with the exception
    """
    raise NameError(message)
